src/Repository.php

```diff
@@ -2,9 +2,19 @@
 namespace Jalno\AutoDiscovery;
 
 use Laravel\Lumen\Application;
-use Jalno\Lumen\Contracts\IAutoDiscovery;
+use Jalno\Lumen\Contracts;
 
-class Repository implements IAutoDiscovery
+/**
+ * @phpstan-type ComposerFile array{
+ *  "name": string,
+ *  "version": string,
```

Comment:
Why do you care about `name`, `version` and `description`?

Reply:
I removed them.